cpu/cortexm_common/include/core_cm0.h

```diff
@@ -39,8 +39,8 @@
  #pragma system_include  /* treat file as system include file for MISRA check */
 #endif
 
-#ifndef __CORE_CM0_H_GENERIC
-#define __CORE_CM0_H_GENERIC
+#ifndef CORE_CM0_H_GENERIC
+#define CORE_CM0_H_GENERIC
```

Comment:
oh, and all CMSIS headers `cpu/cortexm_common/include/core_*.h` must remain untouched.